### PR TITLE
Add go.mod

### PR DESCRIPTION
**Pull request by r2k1**
Setup go modules.

Without it `github.com/go-errors/errors`,  `github.com/pborman/uuid` and `github.com/smartystreets/goconvey` are propagated as indirect dependencies.

It will probably require an update to the travis configuration. But for me, it doesn't seem like it's doing anything useful.

### CODE REVIEW

**Review by izzi-salmon — CHANGES_REQUESTED** [review]
Hi there, thank you for your contribution.

Changes look good, please just increase the raygun4go package version to 1.2.1.

**Comment by r2k1** [comment]
Upgraded to 1.2.0

I think you meant it

File: version.go [hunk 1]
```diff
@@ -1,4 +1,4 @@
 package raygun4go
 
 // the version of raygun4go
-const packageVersion string = "1.1.1"
+const packageVersion string = "1.2.0"
```
Comment (izzi-salmon) [inline version.go:4]: Yes 1.2.0, thanks for catching that.

**Review by izzi-salmon — APPROVED** [review]
Thanks for that. Good to go 👍

**Pull request merged**